### ISSUE TITLE
Error on creating users (v0.23.0)

### ISSUE THREAD

**Issue by robindirksen1** [picqer/moneybird-php-client#198]
The error: `Error 422: {"error":{"company_name":[", first name and last name cannot be empty"]}} ({"options":[]})`

This happens when I want to create a new contact in moneybird.

This happens with the v0.23.0 version released 8 days ago.

**Comment by vjanssens** [comment]
I believe I'm running into a similar issue, but when creating invoices:

```
Error 422: {"error":{"contact_id":["cannot be empty"]}} 
```

It seems as if this piece of code (https://github.com/picqer/moneybird-php-client/blob/master/src/Picqer/Financials/Moneybird/Model.php#L215-L222) isn't properly working.

When you put the following piece of code at the beginning of `isAttributeDirty()`, you'll see the `in_array` returns false because `$attributeName` is a string and `$this->attribute_changes` is a nested array.

```php
dd($attributeName, $this->attribute_changes, in_array($attributeName, $this->attribute_changes));
```

![afbeelding](https://user-images.githubusercontent.com/2340997/77250351-c746b880-6c47-11ea-9be9-923f05bb4fc9.png)

[Comparing v0.23.0 with the previous version](https://github.com/picqer/moneybird-php-client/compare/v0.22.1...v0.23.0#diff-5c8febb556328e093a2c9ffbfffde7f5R303-R306), this behaviour was introduced somewhere in between.

**Comment by robindirksen1** [comment]
If you change this line: 
`if (in_array($attributeName, $this->attribute_changes)) {`
https://github.com/picqer/moneybird-php-client/blob/master/src/Picqer/Financials/Moneybird/Model.php#L217

To: 
`if (array_key_exists($attributeName, $this->attribute_changes)) {`

Or something else:
`if (isset($this->attribute_changes[$attributeName])) {`

Does that work for you?

**Comment by robindirksen1** [comment]
@vjanssens can you test PR #199 if this fixes your issue?

**Comment by robindirksen1** [comment]
@stephangroen can you take a look at this, I think this is a high priority issue for all users.

**Comment by vjanssens** [comment]
@robindirksen1 #199 solves the issue. Thank you 👍

**Comment by stephangroen** [comment]
Thank you for your quick actions @robindirksen1 👍

**Comment by stephangroen** [comment]
Released your fix in v0.23.1
